### ISSUE TITLE
img alt

### ISSUE THREAD

**Issue by ghost** [tck121189/prj-rev-bwfs-tea-cozy#4]
https://github.com/tck121189/prj-rev-bwfs-tea-cozy/blob/3bb0c2f1211125cd8465906d002aa1619133a1dd/Tea%20Cozy/index.html#L36-L52

Make sure to give all  of your `img` tags a descriptive `alt` attribute, just in case your website is not able to load it! 

Doing so will help improve the accessibility of your website when it's visited by visually impaired users on screen readers-- as well as any other time that your website is just unable to render images.
